.circleci/cimodel/data/simple/bazel_definitions.py

```diff
@@ -0,0 +1,67 @@
+DOCKER_IMAGE = "/".join([
+    "308535385114.dkr.ecr.us-east-1.amazonaws.com",
```

Comment:
Can this be moved to `utils/constants.py` or something? Because the same URL is repeated in every other file

Reply:
Agreed, I think it makes more sense to put it in a constants file, probably as a variable named `DOCKER_REGISTRY`.

On that note I think that the tag can also be moved to a `DOCKER_TAG` variable.